edge/services/helloSecretWorld/README.md

```diff
@@ -60,8 +60,7 @@ The following steps will show you how to use the OpenHorizon CLI to create a sec
 2. Register your edge node with OpenHorizon to use the hello-secret pattern:
 
   ```bash
-  export SERVICE_NAME=ibm.hello-secret
-  hzn register -p IBM/pattern-$SERVICE_NAME-$(hzn architecture) -s $SERVICE_NAME --serviceorg IBM
```

Comment:
Is this the change responsible for
> addressed some issue in helloSecretWorld related to only have one all arch pattern

?

Reply:
That is correct